### PR TITLE
Section/services2

### PR DESCRIPTION
**Pull request by whiteblackswan**
https://candid-crisp-121038.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,15 +51,118 @@ <h4 class="section__subtitle">We are a professional digital agency that
...
+
+		<section class="services">
+			<div class="container">
+				<h4 class="title">Our Services</h4>
+				<h2 class="subtitle">We Provide What You Need</h2>
+				<div class="services__wrapper ">
+					<div class="services__item">
+						<img src="icons/FigmaLogo.png" alt="figma">
+						<div class="flex-item__subtitle">UI/UX Designer</div>
+						<div class="subtitle__second">We are ready to make your website more friendly and efficient in the
```
Comment (Dmitruk89) [inline index.html:122]: according to the BEM these two elements should have classes: "services__item__title" and "services__item__description" these classnames will reflect the actual displays of these elements.

File: sass/_services.sass [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+		font-size: 24px
+		line-height: 33px
+		text-align: center
+		color: $maim-black
+	.subtitle__second
+		padding-top: 16px
+		font-weight: 400
+		font-size: 18px
+		line-height: 27px
+		text-align: center
```
Comment (Dmitruk89) [inline sass/_services.sass:35]: and here

File: index.html (hunk 1 above)
Comment (whiteblackswan) [reply index.html:122]: I got bugs, if renamed this.
lO7bk7VqLU

File: sass/_services.sass [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	align-items: center
+	gap: 54px 90px
+	align-content: start
+.services__item
+	width: 289px
+	height: 258px
+	display: flex
+	flex-direction: column
+	justify-content: center
+	align-items: center
```
[the remark this answers is not in the document]
Comment (whiteblackswan) [reply sass/_services.sass:19]: I don't see the need for that, everything working right. If I'll change it not make sense.
8GilYwkTAV

File: sass/_services.sass [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+	background-color: $main-white
+	img
+		padding-bottom: 5px
+	.flex-item__subtitle
+		padding-top: 20px
+		font-weight: 600
+		font-size: 24px
+		line-height: 33px
+		text-align: center
```
[the remark this answers is not in the document]
Comment (whiteblackswan) [reply sass/_services.sass:28]: I fixed it.

File: sass/_services.sass (hunk 2 above)
Comment (whiteblackswan) [reply sass/_services.sass:35]: I fixed it.

**Pull request merged**